src/wallet/load.cpp

```diff
@@ -109,10 +109,21 @@ void StopWallets()
 void UnloadWallets()
 {
     auto wallets = GetWallets();
+    std::string loaded_wallets;
     while (!wallets.empty()) {
         auto wallet = wallets.back();
+
+        // Write the wallet name to rw conf
+        if (!loaded_wallets.empty()) {
+            loaded_wallets += ",";
+        }
+        loaded_wallets += wallet->GetName();
```

Comment:
What if the wallet name has a comma? Would prefer to see rwconf extended to multiple `wallet=` fields...

Reply:
So would I. I would prefer that #11082 implements it that way instead of requiring me to change it as I am not familiar with that code.